src/modules/configurator/hooks/useCarImageSlide.tsx

```diff
@@ -0,0 +1,42 @@
+import { useRef, useState, useEffect } from "react";
+import { fetchCarImagesByColorAndVariant } from "../../../services/API_configurations";
+import { userConfigurationState } from "../state";
+import { useRecoilValue } from "recoil";
+
+export function useCarImageSlide() {
+  const [configImages, setConfigImages] = useState<string[]>();
+  const currentUserConfiguration = useRecoilValue(userConfigurationState);
+
+  const paginationBackRef = useRef(null);
+  const paginationNextRef = useRef(null);
+
+  useEffect(() => {
+    if (currentUserConfiguration) {
```

Comment:
Our practice is first to check the negative one.
```
    if (!currentUserConfiguration) return;
    ...
```

Reply:
Noted and fixed.